tests/foreman/ui/test_classparameters.py

```diff
@@ -748,14 +1078,49 @@ def test_negative_create_matcher_attribute_priority(self):
             1.  The YAML output has the value only for step 5 matcher.
             2.  The YAML output doesn't have value for fqdn/host matcher.
 
-        :caseautomation: notautomated
+        :CaseLevel: Integration
 
         :CaseImportance: Critical
         """
+        sc_param = self.sc_params_list.pop()
+        override_value = gen_string('alphanumeric')
+        override_value2 = gen_string('alphanumeric')
+        with Session(self.browser) as session:
+            set_context(session, org=ANY_CONTEXT['org'])
+            self.sc_parameters.update(
+                sc_param.parameter,
+                self.puppet_class.name,
+                override=True,
+                default_value=gen_string('alpha'),
+                matcher_priority='domain\nhostgroup\nos\nfqdn',
+                matcher=[
+                    {
+                        'matcher_attribute': 'fqdn={0}'.format(self.host.name),
+                        'matcher_value': override_value
+                    },
+                    {
+                        'matcher_attribute': 'domain={0}'.format(
+                            self.domain_name),
+                        'matcher_value': override_value2
+                    }
+
+                ]
+            )
+            self.sc_parameters.click(self.sc_parameters.search_sc_params(
```

Comment:
Yeah, we can introduce that small improvement which is not so necessary (as you need to pass list of matchers, to get their value representation) as it will add less visibility, but from code perspective it is better, so thanks